src/csv.cr

```diff
@@ -208,7 +210,7 @@ class CSV
     @headers || raise(Error.new("Headers not requested"))
   end
 
-  # Invokes the block once for each row in this CSV, yielding `self`.
+  # Invokes the block once for each row in this CSV, yielding `nil`.
```

Comment:
That's an incorrect change.

This method could technically be removed because `Iterator#each` implements basically the same.

Reply:
since Iterator has been removed from CSV is this still relevant?